### DISCUSSION TITLE
pxe name to long

### DISCUSSION THREAD

**Discussion by cchalifo999** [ipxe/ipxe#1129] [General]
when i truy to net boot befor the man menu i get the filename is to long

**Comment by mcb30** [comment]
You will need to provide much more information if you want anyone to be able to help you.  A screenshot might be a good start.

**Reply by cchalifo999** [reply]
screen shot of pxe booting ![PXL_20240123_130306499 MP](https://github.com/ipxe/ipxe/assets/2941510/44673d4b-33d1-4f74-98d0-e2b5f99483ac)
here is the screen shot of the pxe error 

screen shot dhcp server config 
![Screenshot_2024-01-23_08-24-52](https://github.com/ipxe/ipxe/assets/2941510/c9c2883c-8a54-474c-a4b6-e1384529a34b)

**Reply by cchalifo999** [reply]
the tftp is woking fine i am use netboot.xyz as my tftp server working fine and pfsense as y dhcp sertver

**Reply by NiKiZe** [reply]
Your legacy pcbios implementation might make a call to the TFTP service which then responds with error.

Check your TFTP servers logs, and/or use tcpdump/wireshark to figure out what your client asks for and what your server responds with.

You might also want to search for those error codes.

**Comment by mcb30** [comment]
Those error message are coming from the Realtek PXE ROM and are nothing to do with iPXE.

**Reply by cchalifo999** [reply]
pfsens has 2 dchp server Kea DHCP is the newer on that is supported and the old one is ISC DHCP witch is depard (Deprecated)
when the dhcp server is setto ISC DHCP (Deprecated) on pfsense the netboot works fine
